docs/dev/framework/templates/getting-started.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "1. Getting started"
+description: How to quickly get going with the most typical Twig use cases.
+aliases:
+
+- /framework/templates/quick-reference/
+
+---
+
+If you are already familiar with writing Twig templates, you might want to skip this part and directly head over to the
+[next chapter](../architecture). For everyone else, here comes a quick *Twig 101*.
+
+
+## Why Twig in Contao?
+Twig is the standard way to write templates in Symfony. And that is for a reason: It features a wide range of powerful
+methods to structure and reuse templates, has an easy-to-use syntax to access objects, helpers to transform data,
+built-in whitespace control, string interpolation features, macros, and — really — a ton more…
+
+But ultimately, the biggest selling point for us is security. Twig embraces *output encoding*, something that we
+desperately try to move to for a long time. That is, why Twig has been made a core requirement and will replace our old
+existing template system and also why you should familiarize yourself with it.
+
+## Learning the syntax
+Twig templates have their own syntax, but don't be afraid, you'll quickly find your way. Here, is an example of a PHP
+template, that was translated into an equivalent Twig template. (In case you're new to Contao, you might want to directly
+focus on the right side and ignore the old stuff on the left. 😉)
+
+<div style="display:grid;grid-template-columns:1fr 40px 1fr;margin:-20px 0">
```

Comment:
I think we should introduce a CSS class here - so that we can also make it responsive (will get squished too much on mobile 🙃).

Reply:
I factored things out into a separate CSS class, made it responsive, and also applied it to another place in ad2d24757ece7fe47ed3d779ed42a586e26e6529.

Usage looks like this now, the arrow gets inserted via a pseudo element:

```md
<div class="from-to">
  <div>From part</div>
  <div>To part</div>
</div>